src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js

```diff
@@ -10,7 +10,7 @@ export const item = Object.assign(
     format(g) {
       const guesstimateType = this.guesstimateType(g)
       const [low, high] = this._numbers(g.text)
-      return {guesstimateType, low, high }
```

Comment:
I find this quite strange.  If someone estimates, [0.000001,0.00002], would that return as much precision as [5,7]?